scripts/elemental-airgap.sh

```diff
@@ -343,10 +343,13 @@ build_os_channel() {
     local channel_repo
 
     log_info "Creating OS channel"
-    get_chart_val channel_img "channel.repository"
+    get_chart_val channel_img "channel.image"
     get_chart_val channel_tag "channel.tag"
     get_chart_val channel_repo "registryUrl"
 
+    # some images could already have the registry URL, remove it to avoid duplication
+    channel_img=${channel_img#${channel_repo}/}
```

Comment:
Well, right, good catch!
Now all the channel images should be a "full" URL, i.e.,  prepended with the channel repo.
Anyway, the idea is that the channel image can come from another repo than the other images (i.e., for the Rancher Marketplace, the operator and seedimage container images come from docker.io while the channel image is from registry.suse.com). So, we cannot assume that the registry from the channel.image is registryUrl.
Anyway, this PR fixes the script to behave as it was before, so is good as it is, we will improve it later 👍🏼